applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_dynamic_U_Pw_scheme.hpp

```diff
@@ -289,6 +289,30 @@ class NewmarkDynamicUPwScheme : public NewmarkQuasistaticUPwScheme<TSparseSpace,
         KRATOS_CATCH( "" )
     }
 
+    void CalculateRHSContribution(
+        Condition& rCurrentCondition,
+        LocalSystemVectorType& RHS_Contribution,
```

Comment:
Shouldn't this variable name be prefixed with `r` to indicate that it is a reference?
```suggestion
        LocalSystemVectorType& rRHS_Contribution,
```